stglib/tests/data/glob_att1123A_msl.txt

```diff
@@ -0,0 +1,27 @@
+    SciPi; N. Ganju
```

Comment:
Ensure this and all other new test data files are stored using Git LFS and aren't just regular files tracked in Git.

Reply:
Done. Removed files added to stglib/tests/data folder, installed latest version of Git LFS (3.2.0), and tracked files to be added to this folder. added and committed these changes. Verified that files were in  .gitattributes file with filter=lfs.